lib/cmd/fh3/notifications/list.js

```diff
@@ -0,0 +1,50 @@
+/* globals i18n */
+var common = require('../../../common.js');
+var fhreq = require("../../../utils/request");
+var util = require('util');
+
+module.exports = {
+  'desc' : i18n._('List notifications of a RHMAP App'),
+  'examples' : [{
+    cmd : 'fhc notifications list --app=<app> --env=<environment> --audit=<true>',
+    desc : i18n._('List notifications of the <app>')}],
+  'demand' : ['app', 'env'],
+  'alias' : {
+    'app':'a',
+    'env':'e',
+    'audit':'au',
+    'json': 'j',
+    0 : 'app',
+    1 : 'env',
+    2 : 'audit',
```

Comment:
In unix command standards positional parameters have special meaning and are different than command modifiers. 
`cp -Rf source target` 
-Rf is modifier. 

So then what we proposing here is `cp  source target true false` 
Imagine what will happen when some modifiers may cause havoc e.g. `rm -Rf` :P

Let's not talk about it, but I will truly recommend to research command line tools patterns and follow yargs docs :D We trying to improve things - not just move them.